recipes/OpenMiChroM/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "OpenMiChroM" %}
+{% set version = "0.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/OpenMiChroM-{{ version }}.tar.gz
+  sha256: 835ed034621994bde1522760f4601b82721f9f7ad065c3caf1a9a165d15af2b8
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - CLINAME=OpenMiChroM._cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - h5py
+    - numpy
+    - pandas
+    - python >=3.6
+    - scikit-learn
+    - scipy
+    - six
+    - openmm
+
+test:
+  imports:
+    - OpenMiChroM
+    - tests
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://ndb.rice.edu/Open-MiChroM
+  summary: Open-Michrom lib for chromosome simulations
+  license: MIT
+  license_file: LICENSE
```

Comment:
Please add a `dev_url` and a `doc_url` for the package.

Reply:
the dev_url and a doc_url for the package are add.